server/docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  mongodb:
+    image: mongo
+    ports:
+      - 27017:27017
+  redis:
+    image: redis
+    ports:
+      - 6379:6379
```

We can add volume to the `mongodb` service so that our data can persist if the container restarts.
```suggestion
version: '3'
services:
  mongodb:
    image: mongo
    ports:
      - 27017:27017
    volumes:
      - "mongodb-data:/data/db"
  redis:
    image: redis
    ports:
      - 6379:6379
      
volumes:
  mongodb-data: {}
```